source/stylesheets/site.css.scss

```diff
@@ -364,7 +354,8 @@ i.red{
   }
 }
 
-.blog_header {
-  padding-top: 1em;
-  text-align: center;
+.press_release_column {
+  article {
```

Comment:
Its not best practice to modify an elements style even in a descendant selector situation like this. I understand the `press_release_column` is a SMACSS module style but I would prefer for compatibility to define a descendant class instead of the element using the module prefix like this `.press_release_column--article`

Then you can add this class to the article wrapper.